src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -1,12 +1,13 @@
 import Container from '@mui/material/Container'
 import CompleteProfileBlock from '~/components/complete-profile/CompleteProfileBlock'
 import { profileItems } from '~/components/profile-item/complete-profile.constants'
+import VideoPresentation from '~/containers/tutor-profile-page/video-presentation/VideoPresentation'
 
 const TutorProfile = () => {
   return (
-    <Container data-testid='tutorHome' style={ { flex: 1 } }>
-      TutorProfile Page Placeholder
+    <Container data-testid='tutorProfile' style={ { flex: 1, maxWidth: '1128px', margin: '0 auto' } }>
```

Comment:
maxWidth: '1128px', margin: '0 auto' is no needed for container

Reply:
also, remove this data-testid

Reply:
> maxWidth: '1128px', margin: '0 auto' is no needed for container

without _maxWidth_ being set here, my VideoBox Component doesn't render properly

Reply:
> > maxWidth: '1128px', margin: '0 auto' is no needed for container
> 
> without _maxWidth_ being set here, my VideoBox Component doesn't render properly

why? It should be adaptive

Reply:
Fixed